.github/workflows/check-backport.yml

```
@@ -0,0 +1,37 @@
+name: Stable Backport Check
+on:
+  issue_comment:
+  pull_request_target:
+    types:
+      - labeled
+
+jobs:
+  check-backport:
+    name: Check Backport
+    if: ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') && github.event.comment.author_association == 'MEMBER' ) || github.event.label.name == 'stable backport'
```

note: personal prefference 
```suggestion
    if: ${{ ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') && github.event.comment.author_association == 'MEMBER' ) || github.event.label.name == 'stable backport' }}
```